config/routes.rb

```diff
@@ -1,11 +1,15 @@
 Rails.application.routes.draw do
   get '/auth/slack/callback' => 'session#slack_callback'
 
-  root 'pages#index'
+  root 'pre_built_pages#top'
+  get '/wip_term' => 'pre_built_pages#wip_term'
+  get '/thesis' => 'pre_built_pages#thesis'
+
   scope :pages do
-    get   '*path/edit' => 'pages#edit', as: :edit_page
+    get '/' => 'pages#index', as: :pages
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.